test_unstructured_ingest/expected-structured-output/confluence/testteamsp/1605942.json

```diff
@@ -0,0 +1,142 @@
+[
+  {
+    "type": "Title",
+    "element_id": "307afee17dac4c598e361c095338decd",
+    "metadata": {
+      "data_source": {},
+      "filetype": "text/html",
+      "page_number": 1
+    },
+    "text": "Copy and paste this section for each week."
+  },
+  {
+    "type": "Title",
+    "element_id": "b980a145c5e8c9e233a0643366ba520a",
+    "metadata": {
+      "data_source": {},
+      "filetype": "text/html",
+      "page_number": 1
+    },
+    "text": "Win"
+  },
+  {
+    "type": "ListItem",
+    "element_id": "e3b0c44298fc1c149afbf4c8996fb924",
+    "metadata": {
+      "data_source": {},
+      "filetype": "text/html",
+      "page_number": 1
+    },
+    "text": ""
+  },
+  {
+    "type": "ListItem",
+    "element_id": "e3b0c44298fc1c149afbf4c8996fb924",
+    "metadata": {
+      "data_source": {},
+      "filetype": "text/html",
+      "page_number": 1
+    },
+    "text": ""
+  },
+  {
+    "type": "ListItem",
+    "element_id": "e3b0c44298fc1c149afbf4c8996fb924",
+    "metadata": {
+      "data_source": {},
+      "filetype": "text/html",
+      "page_number": 1
+    },
+    "text": ""
+  },
+  {
+    "type": "Title",
+    "element_id": "aecc044c7725a6555114285dc28fe2d1",
+    "metadata": {
+      "data_source": {},
+      "filetype": "text/html",
+      "page_number": 1
+    },
+    "text": "Needs input"
+  },
+  {
+    "type": "ListItem",
+    "element_id": "e3b0c44298fc1c149afbf4c8996fb924",
+    "metadata": {
+      "data_source": {},
+      "filetype": "text/html",
+      "page_number": 1
+    },
+    "text": ""
+  },
+  {
+    "type": "ListItem",
+    "element_id": "e3b0c44298fc1c149afbf4c8996fb924",
+    "metadata": {
+      "data_source": {},
+      "filetype": "text/html",
+      "page_number": 1
+    },
+    "text": ""
+  },
+  {
+    "type": "ListItem",
+    "element_id": "e3b0c44298fc1c149afbf4c8996fb924",
+    "metadata": {
+      "data_source": {},
+      "filetype": "text/html",
+      "page_number": 1
+    },
+    "text": ""
+  },
+  {
+    "type": "Title",
+    "element_id": "9d3cab2b5efed4eaef42a707dbc813da",
+    "metadata": {
+      "data_source": {},
+      "filetype": "text/html",
+      "page_number": 1
+    },
+    "text": "Focus"
+  },
+  {
+    "type": "ListItem",
+    "element_id": "e3b0c44298fc1c149afbf4c8996fb924",
+    "metadata": {
+      "data_source": {},
+      "filetype": "text/html",
+      "page_number": 1
+    },
+    "text": ""
+  },
+  {
+    "type": "ListItem",
+    "element_id": "e3b0c44298fc1c149afbf4c8996fb924",
+    "metadata": {
+      "data_source": {},
+      "filetype": "text/html",
+      "page_number": 1
+    },
+    "text": ""
```

Comment:
what was the deal with these empty string list items?

Reply:
They correspond to the empty rows in the Confluence template docs, which are formed by default (image attached)

We have a few options here:

- Modify these documents to fill the rows so the text fields are not empty
- Delete this kind of template documents and create a few more custom test documents to replace them
- Leave them as they are

I think we can leave them as is, if it won't cause confusion. We have other documents with bulkier text. But if it'd (deceivingly) look like a bug in the future, it might be good to replace them now to avoid confusion later.



![image](https://github.com/Unstructured-IO/unstructured/assets/39141206/c2fed3de-298c-42c3-a067-0f9ee20b7022)

Reply:
yep, totally makes sense and agreed!